taskvine/src/manager/vine_manager.c

```diff
@@ -4682,7 +4687,7 @@ struct vine_task *send_library_to_worker(struct vine_manager *q, struct vine_wor
 	/* If watch-library-logfiles is tuned, watch the output file of every duplicated library instance */
 	if (q->watch_library_logfiles) {
 		char *remote_stdout_filename = string_format(".taskvine.stdout");
-		char *local_library_log_filename = string_format("library-%d.debug.log", q->duplicated_libraries);
+		char *local_library_log_filename = string_format("library-%d.debug.log", q->next_library_id++);
```

Comment:
Hmm, I think this will be confusing because the library ids don't look any different than task ids.

How about just use the task_id in the name of the debug file?

That will make it easy to align library debug file names with other events about that library task in the manager's debug log.